plugin.xml

```diff
@@ -30,7 +30,7 @@
 
   <dependency id="cordova-plugin-whitelist" />
   <dependency id="cordova-plugin-device" />
-  <dependency id="cordova-plugin-push" url="https://github.com/forcedotcom/phonegap-plugin-push" />
+  <dependency id="phonegap-plugin-push" />
```

Comment:
I switched to the vanilla plugin for now. If my tests reveal issues, I'll switch to the fork and apply changes there.